src/assets/i18n/cy.json

```diff
@@ -267,6 +267,26 @@
         "E15": "Goleuadau niwl cefn",
         "E16": "Wedi'u dipio i'r brif olau",
         "E17": "Goleuadau brêciau"
+      },
```

Comment:
Should you maybe add [CY] to these?

Reply:
Not sure what you mean here, can you give me an example?